src/autogen_utils.py

```diff
@@ -3,6 +3,15 @@
 from autogen.agentchat.contrib.capabilities.agent_capability import AgentCapability
 from teams.typing import Typing as TeamsTyping
 from botbuilder.core import TurnContext
+from conversation_state import ConversationState, ChatContext
+from state import AppTurnState
+from botbuilder.schema import Activity, ActivityTypes, Attachment
+from abc import ABC, abstractmethod
+
+class StopableAgentCapability(AgentCapability, ABC):
```

Comment:
what's a "stopable" agent capability?

Reply:
One that can be stopped (has a method called stop()). (Missed a p which I'll update).